### DISCUSSION TITLE
Keeping files in the same drive when moving between folders?

### DISCUSSION THREAD

**Discussion by ghost** [trapexit/mergerfs#1068] [Q&A]
Right now I have `category.create=epmfs`, the default. If I change it to `category.create=epall` so that directories are created in all drives, when I move a file between folders will it get moved to the folder in the same drive or will it still move it to the one with the most free space? I mean do I need to do anything else to move files between folders in the same drive?

**Comment by trapexit** [comment]
By "move" you mean rename? Can you explain exactly what you're looking for? Making func.mkdir=epall means it will create a directory on all branches which have that path. Has nothing to do with how rename behaves. The rename and func.create policy + ignorepponrename do.
